test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -46,8 +49,12 @@
     String prefix = "";
     String retention = "1000";
 
+    EventLoopGroup bossGroup = null;
+    EventLoopGroup workerGroup = null;
+    EventLoopGroup clientGroup = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651477)